### PR TITLE
PARQUET-2478: Update README with link to parquet website

### PR DESCRIPTION
**Pull request by alamb**
You can see the rendered version of this README here:

https://github.com/alamb/parquet-mr/tree/alamb/website_link?tab=readme-ov-file#parquet-mr-

### Jira

- [x] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references
  them in the PR title. For example, "PARQUET-1234: My Parquet PR"
    - https://issues.apache.org/jira/browse/PARQUET-XXX
    - In case you are adding a dependency, check if the license complies with
      the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason: No code is changed

### Commits

There is a single commit with a self explanatory description

- [x] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines
  from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    1. Subject is limited to 50 characters (not including Jira issue reference)
    1. Subject does not end with a period
    1. Subject uses the imperative mood ("add", not "adding")
    1. Body wraps at 72 characters
    1. Body explains "what" and "why", not "how"

### Style
There are no code changes
- [x] My contribution adheres to the code style guidelines and Spotless passes.
    - To apply the necessary changes, run `mvn spotless:apply -Pvector-plugins`

### Documentation
This PR has no java code changes, only markdown
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,19 @@
 Parquet MR [![Build Status](https://github.com/apache/parquet-mr/workflows/Test/badge.svg)](https://github.com/apache/parquet-mr/actions)
 ======
 
-Parquet-MR contains the java implementation of the [Parquet format](https://github.com/apache/parquet-format).
-Parquet is a columnar storage format for Hadoop; it provides efficient storage and encoding of data.
-Parquet uses the [record shredding and assembly algorithm](https://github.com/julienledem/redelm/wiki/The-striping-and-assembly-algorithms-from-the-Dremel-paper) described in the Dremel paper to represent nested structures.
+This repository contains a Java implementation of [Apache Parquet](https://parquet.apache.org/)
+
+Apache Parquet is an open source, column-oriented data file format
```
Comment (alamb) [inline README.md:25]: This is the same wording from https://github.com/apache/parquet-site/pull/59

**Review by vinooganesh — APPROVED** [review]
Looks good to me!

**Review by wgtmac — APPROVED** [review]
LGTM. Thanks!

**Review by julienledem — APPROVED** [review]
Lgtm!

**Pull request merged**